SimpleTCP/SimpleTcpClient.cs

```diff
@@ -9,7 +9,20 @@
 
 namespace SimpleTCP
 {
-	public class SimpleTcpClient : IDisposable
+    public interface ISimpleTcpClient : IDisposable
+    {
+        byte Delimiter { get; set; }
+        Encoding StringEncoder { get; set; }
+        bool AutoTrimStrings { get; set; }
+        SimpleTcpClient Connect(string hostNameOrIpAddress, int port);
+        SimpleTcpClient Disconnect();
+        TcpClient TcpClient { get; }
```

Comment:
As long as .NET's TcpClient is not interfaced, ISimpleTcpClient won't help to mock Connected property.